src/features/import/utils/parseDate.ts

```diff
@@ -9,14 +9,11 @@ dayjs.extend(customParseFormat);
 const placeInCorrectCentury = (date: Date) => {
   const thisYear = new Date().getFullYear();
 
-  //This cutoff is to calculate a date window of 100 years,
-  //so we can decide which century a 2-digit year belongs to.
-  //It seems unlikely users will import people who are older than 110 years of age.
-  const cutoff = thisYear - 110;
-
+  //This cutoff is to calculate which century a 2-digit year belongs to.
+  //It seems unlikely users will import people who are younger than 10 years of age.
```

Comment:
Reading this comment, I realize that we have fixated on age. But a lot of dates are not referring to a person's age, but their join date, date of last activity or things like that. Those are all _more_ likely to be within the last 10 years than 100 years ago.

I wonder what we should do about this? Maybe not solve it now, but note somewhere that this requires some more thinking and potentially UI changes to make this behavior optional for the user?

Reply:
Oh lord this is correct.. how about I change so that anything from 00 to current year gets prefixed with 20, and those above will get 19? that means we might have issues with a few 100-year old members, but if like you say it's more common to use this for just dates in general, then the current way will cause way more problems?

Reply:
I think it's hard to make a choice here, because different types of columns (birth dates vs join dates, for example) will need to be treated differently.